ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
+ETC_20150317_000128	$Attacking its legs will not cause it to topple over.
 ETC_20150317_000129	$Green Sakmoli
 ETC_20150317_000130	$Orange Sakmoli
 ETC_20150317_000131	$Black Sakmoli
 ETC_20150317_000132	$Rikaus
 ETC_20150317_000133	$Green Apparition
-ETC_20150317_000134	Ghost affected by the demons tend to change their appearances like this.
+ETC_20150317_000134	$Ghosts affected by Demons often change their appearance in this way.
 ETC_20150317_000135	$Black Apparition
 ETC_20150317_000136	$Spectre
 ETC_20150317_000137	$Black Spectre
-ETC_20150317_000138	Unlike other monsters in Myan Leong Ryu, these monsters often were evil souls even when they were living.
+ETC_20150317_000138	$Unlike other wraith monsters, these monsters were often evil souls even when they were living.
 ETC_20150317_000139	$Giant Spectre
 ETC_20150317_000140	$Spectre of Deceit
 ETC_20150317_000141	$Green Rikaus
 ETC_20150317_000142	$Spectron
 ETC_20150317_000143	$Spectre Master
 ETC_20150317_000144	$Thornball
-ETC_20150317_000145	This floating seed has transmutated, losing its outer fluff for a solid shell.
+ETC_20150317_000145	$This floating seed has transmutated, losing its outer fluff for a solid shell and thorns. However, its floating capability have stayed intact.
 ETC_20150317_000146	$Big Thornball
 ETC_20150317_000147	$Woodspirit
 ETC_20150317_000148	$Blue Woodspirit
 ETC_20150317_000149	$Green Woodspirit
 ETC_20150317_000150	$Woodman
-ETC_20150317_000151	It would be a real blessing if the Woodman couldn't move.
+ETC_20150317_000151	$The fact that the Woodman can't move is a real blessing.
 ETC_20150317_000152	$Blue Woodman
 ETC_20150317_000153	$Gray Woodman
 ETC_20150317_000154	$Red Woodman
 ETC_20150317_000155	$Dark Slime
-ETC_20150317_000156	Its slimy substance allows it to model itself into a prototype of the object it last encountered, thus the slime can exist in a variety of types.
+ETC_20150317_000156	$Although slimes have similar appearance, they vary in species due to difference between the materials they originated from.
 ETC_20150317_000157	$Rusrat
-ETC_20150317_000158	During the times of Zachariel the Great, these monsters were placed to stop intruders. Its name stems from grave robbers who encountered it.
+ETC_20150317_000158	$Starting from the Royal Mausoleum of Zachariel the Great, these guardians were placed in many ruins to stop intruders. Its name stems from grave robbers who encountered it.
 ETC_20150317_000159	$Madakia
-ETC_20150317_000160	Court sorcerers created the Madakia to fend off intruding grave robbers.
+ETC_20150317_000160	$Court wizards created the Madakia to fend off intruders in case they are not mere grave robbers.
 ETC_20150317_000161	$Wheelen
-ETC_20150317_000162	In order to watch over sealed ruins that lacked an overseer, monsters were created with a diversity of magic. The wheelen is one of these monsters.
+ETC_20150317_000162	$The Wheelen is one of the magical creations made to protect sealed ruins without an overseer.
```

Comment:
"The Wheelen is one of many magical creations made to protect sealed ruins that have no overseer stationed."